harvest/api/dummy.py

```diff
@@ -50,7 +48,7 @@ def __init__(
         self.randomness = {}
 
     def start(self) -> None:
-        pass
+        super().start()
```

Comment:
on `start` the streamer will call the `_base` start which starts an infinite loop getting the new data and calling the `main` function.